contrib/cirrus/CIModes.md

```diff
@@ -42,72 +42,33 @@ of this document, it's not possible to override the behavior of `$CIRRUS_PR`.
 
 ## Cirrus Task contexts and runtime modes
 
-### Intended general PR Tasks (*italic*: matrix)
-+ *build*
-+ validate
-+ bindings
```

Comment:
I have removed these parts because I find them very strange they just duplicate knowledge from the cirrus.yml without explanation and this is out of date and was not updated anyway

Reply:
GOOD! Thank you!